CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 6.0.0
+
+- Change elm-verify-examples.json to contain elm root and not source root. In addition it will respect source-directories defined in elm.json [#109](https://github.com/stoeffel/elm-verify-examples/pull/109)
```

Comment:
```suggestion
- Change elm-verify-examples.json to contain elm root and not source root. In addition it will respect source-directories defined in elm.json [#109](https://github.com/stoeffel/elm-verify-examples/pull/109)

    To migrate, replace the "root" key in your config file for a "elm-root" pointing to the folder containing your elm.json file. Typically this will look like this:
    
     ```diff
     - "root": "../src",
     + "elm-root": "..",
     ```
```